frontend/src/app/main/data/workspace/thumbnails.cljs

```diff
@@ -37,14 +38,13 @@
                      (rx/filter (ptk/type? :app.main.data.workspace/finalize-page))
                      (rx/take 1))]
     (if (some? node)
-      ;; Success: we generate the blob (async call)
-      (rx/create
-       (fn [subs]
-         (ts/raf
-          #(.toBlob node (fn [blob]
-                           (rx/push! subs blob)
-                           (rx/end! subs))
-                    "image/png"))))
+      (->> (rx/from (js/createImageBitmap node))
```

Comment:
I guess this should be `rx/of`

Reply:
No, because `createImageBitmap` is a promise